index.html

```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
     </section>
     
 </div>
+    <div class ="worksection">
+        <section id = "worksectionheader">
+        <h1 id= "worksectionheading"> My Recent Works</h1>
+        <img id="sectiondivider" src ="image/Vector (10).png">
+        </section>
+        
+        <section class="workdescription" >
```

Comment:
- [ ] Your project card has red borders on them. I believe you used them during styling and then forgot to remove them afterward. Kindly remove them.